src/applications/veteran-id-card/config.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+module.exports = {
+
+  messages: {
+    VIC002: (<p>We're sorry. We're having trouble finding your records in our system right now. Please go to <a href="https://pint.access.va.gov/accessva/">AccessVA</a> to request a Veteran ID Card.</p>),
```

Comment:
Not sure if this is the link we want to use for AccessVA.

Reply:
I _think_ `pint` is a test environment and we don't need that part of it.

Reply:
They need to update their `robots.txt` 
![screen shot 2018-05-31 at 08 44 03](https://user-images.githubusercontent.com/4998130/40789091-d2610902-64ae-11e8-9652-fd27279edac9.png)